client/my-sites/plugins/plugin-eligibility/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from '../eligibility-warnings';
+
+class PluginEligibility extends Component {
+	static propTypes = {
+		pluginSlug: PropTypes.string,
+		siteSlug: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	getBackUrl = () => {
+		const { pluginSlug, siteSlug } = this.props;
+
+		return `/plugins/${ pluginSlug }/${ siteSlug }`;
+	};
+
+	goBack = () => page( this.getBackUrl() );
+
+	render() {
+		const { translate } = this.props;
+
+		// Mock some errors and warnings until we get real data from the API
```

Comment:
I think it will be easier to just use `AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST` to fetch real data from the API. I added that in testing instruction.